source/hardware/revisions/rev-d/rev-d-bring-up.md

```diff
@@ -79,4 +97,4 @@ The output of LDO `U16` should be 0V since the PicoZed is not installed. This is
 
 *Congratulations!* If you made it this far, you have successfully brought up a new REV D AMDC! At this point, you are ready to compile the firmware required by the Xilinx Zynq-7000 SoC on the PicoZed and program the AMDC.
 
-For comprehensive instructions, see the [](/firmware/xilinx-tools/building-and-running-firmware.md) document.
\ No newline at end of file
+For comprehensive instructions, see the [Building and Running Firmware](/firmware/xilinx-tools/building-and-running-firmware.md) document.
```

Comment:
No need to change, but, without the title explicitly specified, sphinx uses the linked page's title. This is nice since it makes keeping the links in sync easier if the page title is changed.

Reply:
Are you sure? I added a title in there specifically because the link appeared broken when I built the docs without the provided title...